examples/api.py

```diff
@@ -35,15 +31,21 @@
             "description": "bar bar",
             "introduced_at": 5,
         },
-    }
+    },
 }
 
-foo_api.changelog(5, """
+foo_api.changelog(
+    5,
+    """
     Added baz field.
-""")
-foo_api.changelog(4, """
+""",
+)
+foo_api.changelog(
+    4,
+    """
     Added bar field
-""")
+""",
+)
```

Comment:
Can we inline each of these two changelog lines please? Just drop the trailing commas and `s/"""/"/`.